clang/include/clang/AST/ExternalASTSource.h

```diff
@@ -150,6 +150,11 @@ class ExternalASTSource : public RefCountedBase<ExternalASTSource> {
   virtual bool
   FindExternalVisibleDeclsByName(const DeclContext *DC, DeclarationName Name);
 
+  /// Load all the external specialzations for the Decl and the corresponding
+  /// template arguments.
+  virtual void LoadExternalSpecs(const Decl *D,
```

Comment:
I feel `Load` may be a better name. Since from the signature it doesn't find  anything. And if we want consistency, I suggest to rename `FindExternalVisibleDeclsByName ` to `LoadExternalVisibleDeclsByName`.